projectile.el

```diff
@@ -775,6 +775,12 @@ just return nil."
   "Get the symbol at point and strip its properties."
   (substring-no-properties (or (thing-at-point 'symbol) "")))
 
+(defun projectile-generate-prompt-name (cmd make-new)
```

Comment:
I think `cmd` is a bit misleading here - it's actually just a basename. I'd also use `process/comint-buffer` instead of `prompt`.

Reply:
I went with `(defun projectile-generate-process-name (process make-new)` because I tried a bunch of different wordings and couldn't decide. Feel free to push a commit atop or let me know what you think reads best and I can also do the change.